src/components/Footer.vue

```diff
@@ -45,4 +45,9 @@
 export default {
   name: 'Footer',
 }
-</script>
\ No newline at end of file
+</script>
+<style lang="scss">
+h5{
```

Comment:
Always add a space between the `{` and the tag
```suggestion
h5 {
```